__tests__/index.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { computed, defineComponent, nextTick } from 'vue'
+import { useFlags, useFlagsmith, useFlagsmithLoading, useTraits } from '../src/index'
+import { mount } from '@vue/test-utils'
+import flagsmith from 'flagsmith'
+
+jest.mock('flagsmith')
+
+const ChildComponent = defineComponent({
+    template: `
+    <div class="child-component"
+        :test-trait="testTrait">
+        <div v-if="isVisible"
+            class="child-component--feature">
+        </div>
+        <div v-if="isLoading"
+            class="child-component--loading">
+        </div>
+    </div>`,
+    setup() {
+        const { is_visible: isVisibleFeature } = useFlags(['is_visible'])
+        const isVisible = computed(() => isVisibleFeature.value?.value)
+        const { test_trait: testTrait } = useTraits(['test_trait'])
+        const { isLoading } = useFlagsmithLoading()
+
+        return { isVisible, testTrait, isLoading }
+    },
+})
+
+const isVisibleFeatureMock = {
```

Comment:
It would make sense to add a type here, so that the mock actually conforms to the expected type. Something like this:

```ts
const isVisibleFeatureMock: IFlagsmithFeature = {…}
```

Reply:
Yes makes sense. Changed it now.